app/presenters/hyrax/collection_presenter.rb

```diff
@@ -41,17 +41,20 @@ def collection_type
     # Terms is the list of fields displayed by
     # app/views/collections/_show_descriptions.html.erb
     def self.terms
-      [:total_items, :size, :resource_type, :creator, :contributor, :keyword, :license, :publisher, :date_created, :subject,
+      [:total_items, :resource_type, :creator, :contributor, :keyword, :license, :publisher, :date_created, :subject,
        :language, :identifier, :based_near, :related_url]
     end
 
     def terms_with_values
       self.class.terms.select { |t| self[t].present? }
     end
 
+    ##
+    # @param [Symbol] key
+    # @return [Object]
     def [](key)
       case key
-      when :size
+      when size
```

Comment:
I believe this ought to be a symbol, `:size`.

Reply:
Agreed, this should be `:size`